### ISSUE TITLE
E251: false positive for PEP 696 TypeVar defaults

### ISSUE THREAD

**Issue by asottile-sentry** [PyCQA/pycodestyle#1277]
https://peps.python.org/pep-0696/

minimal example:

```python
class C[T: (int, str) = int]:
    pass
```

```console
$ ./venv/bin/flake8 t4.py 
t4.py:1:22: E251 unexpected spaces around keyword / parameter equals
t4.py:1:24: E251 unexpected spaces around keyword / parameter equals
```
